purchase_company_currency/README.rst

```diff
@@ -0,0 +1,55 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
Consider adding a nice README using https://github.com/OCA/maintainer-tools/blob/master/tools/gen_addon_readme.py

Reply:
Will not do. Too old